v3/setup/splunk.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# SPLUNK_FORWARD_SERVER is declard in infrastructure config/secrets.json
+
+source /etc/environment
+
+etcdctl set /images/splunkforwarder "adobeplatform/docker-splunk:latest"
```

Comment:
I would go as far as saying that this can be placed right into the systemd unit and you can just remove this file